discovery-provider/src/app.py

```diff
@@ -550,6 +551,12 @@ def configure_celery(celery, test_config=None):
         cid_metadata_client,
     )
 
+    registry_address = web3.toChecksumAddress(
```

Comment:
should this be in init_contracts above?

Reply:
well I wasn't sure what the init_contract above is used for, but I wanted to organize the code some more and made eth_manager to do that
I think we should remove init_contracts actually